zerver/lib/message.py

```diff
@@ -558,7 +558,8 @@ def build_dict_from_raw_db_row(row: Dict[str, Any]) -> Dict[str, Any]:
                 'reaction_type': row['reaction_type'],
                 'user': {'email': row['user_profile__email'],
```

Comment:
Let's add a comment like `TODO: We plan to remove the user object once clients are updated to just use user_id` here, and also in the `notify` code in `actions.py`, maybe in both places with a reference to this PR.